SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -365,16 +366,16 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	UFUNCTION(BlueprintPure, Category = "Spatial Functional Test")
 	bool WasLoadedFromTakenSnapshot();
 
-	template <typename T>
-	static int GetNumberOfActorsOfType(UWorld* World)
+	template <typename Actor_Type>
+	static SIZE_T CountActors(UWorld* World)
```

Comment:
drive by

Reply:
It's `SIZE_T` due to `Algo::Count` returning `SIZE_T`, right? Just clarifying as I don't really see it elsewhere.

Reply:
yeah other places use `int32` I could change to that.

Reply:
```suggestion
	static int32 CountActors(UWorld* World)
```